pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -648,6 +657,7 @@ monitorLoop:
 				progressWatermark = remainingData
 				lastProgressUpdate = now
 			}
+
 			// check if the migration is progressing
 			progressDelay := now - lastProgressUpdate
 			if progressTimeout != 0 &&
```

Comment:
@ashleyschuett we should probably ignore this when we've switched to post-copy... what do you think?
I guess there is more harm than benefit.

Reply:
I was curious about this as well, once switched to post copy is there a chance the migration could for a time stop progressing but still be okay and should be continued? My concern with ignoring it is then all migration checks are ignored once the migration is switched to post and there's nothing monitoring it to cancel it. However, looking through the implementation from openstack that is also the case, so I gather that doesn't matter?

Reply:
On one hand, you are right, we don't want to ignore all the checks, but the trouble with canceling the migration automatically in the post-copy is that we will lose state and I'm not sure it's helpful.
Perhaps we can log or emit an event instead of canceling?

Reply:
id think that not timing out on progress post copy is probably fine. if the connection is completely severed, my expectation is that libvirt would fail the migration